src/index.js

```diff
@@ -85,7 +85,7 @@ exports.getNumber = (name) => {
   if (code === undefined) {
     throw new Error('Codec `' + name + '` not found')
   }
-  return util.varintUint8ArrayDecode(code)[0]
+  return util.varintUint8ArrayDecode(code)
```

Comment:
It looks like the `util.carintUint8ArrayDecode()` function isn't used anywhere else. I propose removing it com `util.js` completely and directly use `varint.decode(input)` here.